### PR TITLE
language update

### PR DESCRIPTION
**Pull request by ascrnet**
Updated all the Spanish language to version 5.3 and also eliminated two repeated lines that I found in the language configuration.

### CODE REVIEW

**Comment by RespeQt** [comment]
Hello,
Sorry for this long delay as I've been away from Atari since the start of 2021.
I wish I could accept your 5 commit but the number 5695fd9 (Removal of the warning) introduces bugs.
As you can see in the source, there is a comment saying that there is a fall-thru.
If you put a break at the end of the case, we will no longer go to the next case.
I agree that fall-thru are not a best practice. They should be removed but not this way.
Could you change that on your side and make another pull request?
Best Regards

File: printers/atari1020.cpp [hunk 1]
```diff
@@ -419,8 +419,8 @@ namespace Printers
             if (b == '-') {
                 mFirstNegative = true;
                 mAutomataState = AUTOMATA_FIRST_INT;
```
Comment (ascrnet) [inline printers/atari1020.cpp:421]: is reversed because it affects the operation

File: printers/atari1020.cpp [hunk 2]
```diff
@@ -467,8 +467,8 @@ namespace Printers
             if (b == '-') {
                 mSecondNegative = true;
                 mAutomataState = AUTOMATA_SECOND_INT;
-                break;
```
Comment (ascrnet) [inline printers/atari1020.cpp:470]: is reversed because it affects the operation

File: printers/atari1020.cpp [hunk 3]
```diff
@@ -511,8 +511,8 @@ namespace Printers
             if (b == '-') {
                 mThirdNegative = true;
                 mAutomataState = AUTOMATA_THIRD_INT;
-                break;
```
Comment (ascrnet) [inline printers/atari1020.cpp:514]: is reversed because it affects the operation

File: printers/atari1020.cpp [hunk 4]
```diff
@@ -511,8 +511,8 @@ namespace Printers
...
             }
+            break;
             // FALL THRU if '-' is not found
 
         case AUTOMATA_THIRD_INT:
```
Comment (ascrnet) [inline printers/atari1020.cpp:518]: is reversed because it affects the operation

**Review by ascrnet — COMMENTED** [review]
is reversed because it affects the operation

**Comment by ascrnet** [comment]
ok, reverse the 5695fd9 change so you can approve the others.

**Pull request merged**